lib/js/components/NumberInCircle/NumberInCircle.vue

```diff
@@ -36,32 +37,44 @@
 		min-width: $icon-xxl;
 	}
 
-	&.-is1 {
-		color: $color-primary;
-	}
-
-	&.-is2 {
-		color: $color-salad-green;
+	&.-is1,
+	&.-is6,
+	&.-primary {
+		color: $color-primary-text;
+		border-color: $color-primary-border;
```

Comment:
we should get rid of `currentColor` in default state now

Reply:
Good catch, I'm removing all usages. There was only problem in `Modal.vue` - it helps to catch usage old colors.